fundamentals/installation.md

```diff
@@ -45,14 +45,37 @@ If you installed napari with `pip` you can upgrade by calling
 $ pip install napari --upgrade
 ```
 
-## choosing a different backend
-By default, napari uses [PySide2](https://wiki.qt.io/Qt_for_Python). This is what most users will want. 
+## choosing different Qt backends
 
-If you want to use [PyQt](https://www.riverbankcomputing.com/software/pyqt/intro) instead for your backend, you can run these lines after installing napari:
+Most users can safely ignore this section. If, however, you aim to use napari together with software that uses PyQt5, or have some other reason to prefer that Qt backend, read on.
+
+[Qt](https://www.qt.io) is a C++ library that provides unified graphical user interface (GUI) elements and interactivity across platforms and operating systems. Several libraries exist for accessing these elements from Python, which we call in the context of napari a *backend*.
+
+The default napari backend is [PySide2](https://wiki.qt.io/Qt_for_Python), and this is what you get if you type `pip install napari` or `conda install -c conda-forge napari` and you do not have [pyqt5](https://pypi.org/project/PyQt5/) installed. If you wish to use [PyQt5](https://www.riverbankcomputing.com/software/pyqt/intro) as the backend instead, you can install pyqt5 and napari from PyPI like this.
+
+Installing napari from PyPI, using a [pyqt5](https://pypi.org/project/PyQt5/) backend:
+```
+pip install pyqt5
+pip install napari
+```
+
+**Note: The order of installing packages is important, [pyqt5](https://pypi.org/project/PyQt5/) must be installed *before* napari.**
+
+If you would like install napari from the git repository instead, and use [pyqt5](https://pypi.org/project/PyQt5/) instead of [PySide2](https://wiki.qt.io/Qt_for_Python), then you also need to use the pip argument `--no-use-pep517`.
```

Comment:
Can you explain very briefly why you need `--no-use-pep517`, i've never heard of it

Reply:
Pep 517 introduce isolated build system https://www.python.org/dev/peps/pep-0517/. So during installation there is created wheel in isolated virtualenv, which block checking current environment (checking if PyQt5 is already installed). Currently pip decides to use pep517 if pyproject.toml is present on top level (even it does not have `[build-system]` section).